### PR TITLE
Moved privacy page to own controller

### LINKED ISSUE #662: Move privacy page to separate controller
Move privacy page from logix controller to own controller

### PR DESCRIPTION
**Pull request by mohit-003**
Fixes #662 

#### Describe the changes you have made in this pr -
Moved privacy controller from logix controller to seperate controller.

### CODE REVIEW

**Comment by mohit-003** [comment]
@alanjsph Please review my task.

File: app/controllers/privacy_controller.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class PrivacyController < ApplicationController
+  def privacy
```
Comment (ankitkataria) [inline app/controllers/privacy_controller.rb]: We can use the `index` method here.

File: config/routes.rb [hunk 2]
```diff
@@ -55,7 +55,7 @@
   get  '/gettingStarted', to:'logix#gettingStarted'
   get  '/examples', to:'logix#examples'
   get  '/features', to:'logix#features'
-  get  '/privacy', to:'logix#privacy'
+  get  '/privacy', to:'privacy#privacy'
```
Comment (ankitkataria) [inline config/routes.rb]: Let's use `resource` routes here with `only: :index`

**Comment by mohit-003** [comment]
Done @ankitkataria

File: config/routes.rb [hunk 3]
```diff
@@ -55,7 +56,7 @@
...
```
Comment (tachyons) [inline config/routes.rb]: you don't need this route anymore

File: desktop.ini [hunk 4]
```diff
@@ -0,0 +1,4 @@
+[ViewState]
+Mode=
+Vid=
+FolderType=StorageProviderGeneric
```
Comment (tachyons) [inline desktop.ini]: ?

Comment (mohit-003) [reply desktop.ini]: I had changed the folder icon by mistake ,that's why it is generated.

**Pull request merged**